Automattic/jetpack
# Untangling: Add "Appearance > Theme Showcase" menu

Fixes https://github.com/Automattic/dotcom-forge/issues/6523

## Proposed changes:

Registers an "Appearance > Theme Showcase" menu for untangled sites.

Before | After
--- | ---
<img width="338" alt="Screenshot 2024-04-11 at 16 46 46" src="https://github.com/Automattic/jetpack/assets/1233880/e49f0a03-7df6-4385-9c3b-87521c7e598c"> | <img width="334" alt="Screenshot 2024-04-11 at 16 51 34" src="https://github.com/Automattic/jetpack/assets/1233880/f1718df3-dd21-4ea1-98da-5a10d6ba6fd3">


### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- If you don't have an untangled Atomic site:
  - Create or pick a regular Atomic site
  - Go to wordpress.com/hosting-config and select your Atomic site
  - Change the admin interface to wp-admin
  - Go to `<SITE_DOMAIN>/_cli` and run this command: `wp option update wpcom_classic_early_release 1`
- Apply these changes to your untangled Atomic site:
  - Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/)
  - Go to `/wp-admin/admin.php?page=jetpack-beta&plugin=jetpack-mu-wpcom-plugin`
  - Activate this branch
  - Add the following to your `wp-config.php` file: `define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );`
- Make sure there is a new "Theme Showcase" submenu under the "Appearance" submenu
- Make sure it links to the Calypso Theme Showcase
- Apply these changes to a "tangled" Atomic site with the classic admin interface:
  - Go to `<SITE_DOMAIN>/_cli` and run this command: `wp option update wpcom_classic_early_release 0`
  - Go to `/wp-admin/admin.php?page=jetpack-beta&plugin=jetpack`
  - Activate this branch
- Make sure the "Theme Showcase" submenu is not duplicated
